lib/warden/strategies/doorkeeper_strategy.rb

```diff
@@ -0,0 +1,18 @@
+require "warden/strategies/doorkeeper"
+require "warden/action_dispatch_patch"
+
+class Warden::Strategies::Doorkeeper < ::Warden::Strategies::Base
+  include ActionDispatchPatch
+
+  def valid?
```

Comment:
It's should be tested in doorkeeper_auth_test actually. We don't test other methods.